R/component.R

```diff
@@ -21,7 +21,9 @@
 #' - `conv_message`: message accompanying the convergence code.
 #' - `evaluations`: number of function evaluations for optimization.
 #' - `beta_vcov`: estimated variance-covariance matrix of coefficients
-#'      (excluding aliased coefficients).
+#'      (excluding aliased coefficients). For Kenward-Roger and Kenward-Roger-Linear
```

Comment:
```suggestion
#'      (excluding aliased coefficients). For Kenward-Roger
```
since Kenward-Roger-Linear is also one specific Kenward-Roger method.